m4/geany-status.m4

```diff
@@ -27,7 +27,8 @@ EOF
 	# Print a nice top bar
 	# description + ' : ' + value
 	total=`expr $dlen + 3 + $vlen`
-	for i in `seq 1 $total`; do printf '-'; done
+	echo
+	echo "    Build configuration result:"
```

Comment:
Sadly `printf '%-*s' $total '' | tr ' ' -` isn't portable…

But what about this?
```shell
i=0; while test $i -lt $total; do printf '-'; i=`expr $i + 1`; done
```

Reply:
what about echo "----------------------------------------"

thats guaranteed to be portable (and what configure.in had a long time ago :)

Reply:
> what about echo "----------------------------------------"
> 
> thats guaranteed to be portable (and what configure.in had a long time ago :)

That seems like a good choice, imo. Let me know when there's agreement and I'll make the change.